bundles/org.openhab.binding.yamahamusiccast/src/main/java/org/openhab/binding/yamahamusiccast/internal/YamahaMusiccastHandler.java

```diff
@@ -593,6 +619,11 @@ private void updateStateFromUDPEvent(String zoneToUpdate, UdpMessage targetObjec
             updateState(channel, new DecimalType(volumeState));
         }
 
+        if (volumeDbState != -80f) {
+            channel = new ChannelUID(getThing().getUID(), zoneToUpdate, CHANNEL_VOLUMEDB);
+            updateState(channel, new DecimalType(volumeDbState));
```

Comment:
To support UoM here you could use:
```suggestion
            updateState(channel, new QuantityType<>(volumeDbState, Units.DECIBEL));
```